src/Appwrite/Task/Doctor.php

```diff
@@ -0,0 +1,256 @@
+<?php
+
+namespace Appwrite\Task;
+
+use Utopia\App;
+use Utopia\CLI\Console;
+use Appwrite\ClamAV\Network;
+use Utopia\Logger\Logger;
+use Utopia\Storage\Device\Local;
+use Utopia\Storage\Storage;
+use Utopia\Domains\Domain;
+use Utopia\Platform\Action;
+
+class Doctor extends Action
+{
+    public const NAME = 'doctor';
+
+    public function __construct()
+    {
+        $this
+            ->desc('Validate server health')
+            ->callback(fn () => $this->action());
+    }
+
+    public function action(): void
+    {
+        global $register;
```

Comment:
We really need to find a way to replicate dependency injection to other platforms.

Reply:
updated with Dependency injection